composer.json

```diff
@@ -1,24 +1,22 @@
 {
     "name": "jordanbrauer/unit-converter",
-    "type": "component",
     "description": "Convert standard units from one to another with this easy to use, lightweight package",
-    "version": "0.8.7",
```

Comment:
`version` conflicts with the git tag - best to get rid of it, as forgetting to change it will just lead to confusion